### PR TITLE
fix GH actions and pin setuptools version for python 3.6

### PR DESCRIPTION
**Pull request by smoors**
edit: now reproduces problem reported in #82

### CODE REVIEW

File: .github/workflows/pip_install.yml [hunk 1]
```diff
@@ -9,12 +9,16 @@ jobs:
       fail-fast: false
       matrix:
         python: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
+        setuptools-version: ['39.2.0', '59.6.0', '68.2.2']
```
Comment (boegel) [inline .github/workflows/pip_install.yml:12]: Not all of these are probably compatible with all those Python versions...

Ideally we just add one extra combo to cover the fix in #86?

Comment (smoors) [reply .github/workflows/pip_install.yml:12]: indeed, i first wanted to check which combos are ok and which are not.

now that it's clear that it's setuptools `39.2.0` is the problem, we can reduce the number of tests

**Comment by smoors** [comment]
the pip install checks were not working properly:

- they were not actually using different python versions; it was always using python 3.10, the default on ubuntu 22.04.
- the tests (importing the modules) did not actually test the installation, because it was run from the root of the sources

with that fixed, the installation is not working for any version, which is expected because both setup.cfg and pyproject.toml need fixing.

**Comment by boegel** [comment]
@smoors GitHub doesn't realize that the changes proposed here have been merged via #86, so we should just close this then?

**Pull request closed without merging**